src/libraries/System.Net.Requests/src/System/Net/FtpControlStream.cs

```diff
@@ -885,26 +888,28 @@ private static long GetContentLengthFrom213Response(string responseString)
         private DateTime GetLastModifiedFrom213Response(string str)
         {
             DateTime dateTime = _lastModified;
-            string[] parsedList = str.Split(new char[] { ' ', '.' });
-            if (parsedList.Length < 2)
+            Span<Range> parts = stackalloc Range[4];
+            ReadOnlySpan<char> strSpan = str;
+            int count = strSpan.SplitAny(parts, " .", StringSplitOptions.RemoveEmptyEntries);
```

Comment:
Why did you add RemoveEmptyEntries?

Reply:
Just in case, by assuming empty entries could not be part of date/time, but as the original split did not have it, I can remove it

Reply:
```suggestion
            int count = strSpan.SplitAny(parts, " .");
```